source/_components/cover.garadget.markdown

```diff
@@ -43,3 +43,48 @@ Configuration variables:
 
 
 If provided, the **access_token** will be used, otherwise the **username** and **password** will be used to automatically generate an access token at start time.
+
+
+**Example with more detail:**
+<p class='img'>
+<img src='https://dl.dropboxusercontent.com/u/15301912/GitHub/home-assistant.github.io/images/cover_garadget_details.png' />
+</p>
+
+```yaml
+# Related configuration.yaml entry
+cover:
+  platform: garadget
+  covers:
+    190028001947343412342341:
+      access_token: !secret garadget_access_token
+      name:  Garage door
+
+sensor:
+  platform: template
+  sensors:
+    garage_door_status:
+      friendly_name: 'State of the door'
+      value_template: '{{ states.cover.garage_door.state }}'
+    garage_door_time_in_state:
+      friendly_name: 'Since'
+      value_template: '{{ states.cover.garage_door.attributes["time in state"] }}'
```

Comment:
All templates must be embedded with `{% raw %}...{% endraw %}` to avoid displaying empty string after rendering.